src/assets/styles/layout.scss

```diff
@@ -52,6 +52,14 @@
   }
 }
 
+.section-container {
```

Comment:
I think a lot of divs use this style already. Did you visually check that these new attributes look OK in the existing places?

Reply:
It does, indeed. 
If this was used in the components i believe the intention was there to apply this styling. I was surprised it wasn't already global. 
Will give a double check and remove and local instances of this styling.

Reply:
Removed the local instances of this in the next commit 👍